charts/stable/traefik/questions.yaml

```diff
@@ -562,28 +562,38 @@ questions:
                   attrs:
                     - variable: name
                       label: "Name"
+                      description: This is a 3rd party plugin and not maintained by TrueCharts,
+                        for more information go to <a href="https://github.com/packruler/traefik-themepark">traefik-themepark</a>
                       schema:
                         type: string
                         required: true
-                    - variable: app
-                      label: "Application Name"
-                      description: "Lower case, supported list https://docs.theme-park.dev/themes."
+                    - variable: appName
+                      label: App Name
+                      description: Lower case, name of the app to be themed.
+                        Go to <a href="https://docs.theme-park.dev/themes/">https://docs.theme-park.dev/themes/</a> to see supported apps.
                       schema:
                         type: string
                         required: true
-                        default: ""
-                    - variable: theme
-                      label: "Theme Name"
-                      description: "Lower case, supported lists https://docs.theme-park.dev/theme-options and https://docs.theme-park.dev/community-themes"
+                    - variable: themeName
+                      label: Theme Name
+                      description: Lower case, name of the theme to be applied.
+                        Go to <a href="https://docs.theme-park.dev/theme-options/">https://docs.theme-park.dev/theme-options/</a> to see supported themes.
                       schema:
                         type: string
-                        default: ""
-                    - variable: baseUrl
-                      label: "Base URL for theme"
-                      description: "https://theme-park.dev or a self hosted url"
+                        required: true
+                    - variable: enableBaseUrl
+                      label: Enable Custom Base URL
+                      description: Replace `https://theme-park.dev` URL for self-hosting reference.
                       schema:
-                        type: string
-                        default: "https://theme-park.dev"
+                        type: boolean
+                        default: false
+                        show_subquestions_if: true
+                        subquestions:
+                          - variable: baseUrl
+                            label: Base URL
+                            schema:
+                              type: string
+                              required: true
```

Comment:
I believe it's best to remove the enable toggle, have it `required` and set the default to `https://theme-park.dev`.

This will make UI less nested (tbh scale had some lags some time ago with multiple nested keys.)

But besides all that. 

Because our charts are also used by native helm users. having the enabled here 
Will also require to have `enableBaseUrl` in values.yaml.
So  `{{- if $middlewareData.enableBaseUrl }}` evaluates to true or false for native helm users..
Which also adds extra un-needed "complexity".

What do you think?

Reply:
Ah that makes sense. Didn't think about supporting standard helm integrations.